src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,30 @@
+import * as RadixPopover from '@radix-ui/react-popover';
+import clsx from 'clsx';
+import type { FC, ForwardedRef } from 'react';
+import { forwardRef, memo } from 'react';
+import { inter } from 'src/pages/_app';
```

Comment:
Lets rename it to `font` or something else abstract.

Also did you try using `inherit`?

Reply:
`Popover` is mounted into `body` via `Portal`, so `inherit` is not an option here